vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionAboutPage.java

```diff
@@ -45,19 +77,20 @@ public String getTitle() {
         return title;
     }
 
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
     public String getAboutPageText() {
         return aboutPageText;
     }
+    
+    
+    public void setTitle(String title) {
+        this.title = title;
+    }
 
     public void setAboutPageText(String aboutPageText) {
         this.aboutPageText = aboutPageText;
     }
-    
-    /*
+	
+	/*
```

Comment:
getters and setters should be in the order of the variables; also, why is there extra indent here?